server/actions/status-check.js

```diff
@@ -1,24 +1,33 @@
 const _ = require("lodash");
 const lint = require("../lib/utils/lint-code");
 const syntaxCheck = require("../lib/utils/syntax-check");
+const manifest = require("../../manifest");
+
+// Get the default code provided by the manifest
+const index = _.findIndex(manifest.private_settings, item => item.name === "code");
+const defaultCode = manifest.private_settings[index].default;
 
 module.exports = function statusCheck(req, res) {
   const { ship, client } = req.hull;
   const messages = [];
   let status = "ok";
 
   const code = _.get(ship.private_settings, "code", "n/a");
-  if (code === "n/a") {
+  if (code === "n/a" || _.trim(code) === "") {
```

Comment:
I've made sure that there aren't any trailing whitespaces, otherwise the warning wouldn't trigger. Also, comparing the returned code with "n/a" never worked, but I left it in case I missed something.

Reply:
seems good.